ow_lander/scripts/path_planning_commander.py

```diff
@@ -55,10 +57,12 @@ def handle_stow(self, req):
     plan = self.arm_move_group.plan()
     if len(plan.joint_trajectory.points) == 0:
       return False
-    self.trajectory_async_executer.execute(plan.joint_trajectory)
+    self.trajectory_async_executer.execute(plan.joint_trajectory, 
+                                          active_cb=None,
```

Comment:
I think you can safely drop the active_cb
```suggestion
```

Reply:
done locally